ds-caselaw-ingester/tests.py

```diff
@@ -40,7 +43,7 @@
         "parameters": {
             "status": "JUDGMENT_PARSE_NO_ERRORS",
             "reference": "TDR-2022-DNWR",
-            "originator": "FCL",
+            "originator": "TDR",
```

Comment:
As per meeting - worth having another look at this in depth to check whether this is correct!

Reply:
Now I thought about it: yes. This example is supposed to reflect the ordinary case of "something new comes from TDR".